mspray/apps/main/tests/views/test_user.py

```diff
@@ -0,0 +1,66 @@
+"""Test user login and logout module."""
+from unittest.mock import patch
+from django.test import TestCase
+
+from mspray.apps.main.views.user import get_form_users
+
+form_data = {
```

Comment:
I think a Python convention is to capitalise such variable names and pep8 says so as well: https://www.python.org/dev/peps/pep-0008/#constants